test-harness/specs/parameters-ac7.oas2.txt

```diff
@@ -26,7 +26,7 @@ paths:
 ====server====
 mock -p 4010
 ====command====
-curl -i -X POST http://localhost:4010/path -H "Content-Type: application/x-www-form-urlencoded" --data "arr=a,b,c"
+curl -i -X POST http://localhost:4010/path -H "Content-Type: application/x-www-form-urlencoded" --data "arr=a%2Cb%2Cc"
```

Comment:
We can use `--data-urlencode` to let CURL encode the data and keep the test readable

```suggestion
curl -i -X POST http://localhost:4010/path -H "Content-Type: application/x-www-form-urlencoded" --data-urlencode "arr=a,b,c"
```

Reply:
Done, good catch btw. :)